.circleci/config.yml

```diff
@@ -820,6 +831,12 @@ workflows:
       - build_docs
       - black
       - flake8
+      - deploy_reliability:
```

Comment:
this should require `deploy_master`, we want to do it after we successfully publish the wheels.

Reply:
I don't see why this is necessary, we can (and do) install using the commit hash

```
pip install git+https://github.com/DataDog/dd-trace-py.git@<HASH>
```

Reply:
if we do it from a wheel, we can verify that our wheels building process works, and it won't require Cython to be installed in order to build/deploy into the reliability env.

Reply:
Got it, but if we want to test that our wheels work the reliability env is (currently) not the place to do it. We don't test every python version or OS platform. Perhaps we want to investigate adding a job in our CI to test the wheels? cc @majorgreys

Reply:
Also how could we permit deploying non-master, non-staging branches which don't have wheels built?

Reply:
got it. your call here then